cmd/skaffold/app/cmd/debug.go

```diff
@@ -36,6 +36,7 @@ func NewCmdDebug() *cobra.Command {
 		WithDescription("[beta] Run a pipeline in debug mode").
 		WithLongDescription("Similar to `dev`, but configures the pipeline for debugging.").
 		WithCommonFlags().
+		AllowHouseKeepingMessages().
```

Comment:
Can we rename this to `withHouseKeepingMessages()` ?

Reply:
done